presto-main/src/main/java/com/facebook/presto/operator/aggregation/fixedhistogram/FixedDoubleHistogram.java

```diff
@@ -111,34 +128,30 @@ public int getRequiredBytesForSerialization()
 
     public void serialize(SliceOutput out)
     {
-        out.appendByte(FORMAT_TAG)
-            .appendInt(bucketCount)
-            .appendDouble(min)
-            .appendDouble(max)
-            .appendBytes(Slices.wrappedDoubleArray(weights, 0, bucketCount));
+        out.appendInt(bucketCount)
+                .appendDouble(min)
+                .appendDouble(max)
+                .appendBytes(Slices.wrappedDoubleArray(weights, 0, bucketCount));
     }
 
     public long estimatedInMemorySize()
     {
-        return INSTANCE_SIZE + SizeOf.sizeOf(weights);
+        return INSTANCE_SIZE + getRequiredBytesForSerialization();
```

Comment:
Is this method used for memory tracking? Why change `sizeOf(weights)` to `getRequiredBytesForSerialization()`?  `getRequiredBytesForSerialization()` doesn't necessary mean retained size (and can have difference in some cases) right?